spicy/toolchain/src/compiler/visitors/validator.cc

```diff
@@ -571,6 +571,35 @@ struct VisitorPost : public hilti::visitor::PreOrder<void, VisitorPost>, public
         }
     }
 
+    void checkBits(const spicy::type::Unit& u, position_t p, const hilti::node::Set<type::unit::Item>& items,
+                     std::set<ID>* seen_bits) {
+        for ( const auto& item : items ) {
+            if ( auto f = item.template tryAs<spicy::type::unit::item::Field>() ) {
```

Comment:
We don't seem the `template` here as `item` is not a dependent type.

```suggestion
            if ( auto f = item.tryAs<spicy::type::unit::item::Field>() ) {
```